docs/deployment/upgrade.md

```diff
@@ -8,53 +8,60 @@ hide_title: true
 
 After the initial deployment, any future updates to the Cumulus deployment from configuration files, Terraform files (`*.tf`), or modules from a new version of Cumulus can be deployed and will update the appropriate portions of the stack as needed.
 
-## Cumulus versioning
+## Cumulus Versioning
 
-Cumulus uses a global versioning approach, meaning version numbers are consistent across all packages and tasks, and semantic versioning to track major, minor, and patch version (i.e. 1.0.0). We use Lerna to manage versioning.
+Cumulus uses a global versioning approach, meaning version numbers are consistent across all packages and tasks, and semantic versioning to track major, minor, and patch version (e.g., 1.0.0). We use Lerna to manage versioning.
 
-To view the released module artifacts for each Cumulus core version, see the [Cumulus releases page](https://github.com/nasa/cumulus/releases).
+To view the released module artifacts for each Cumulus core version, see the [Cumulus Releases] page.
 
-**IMPORTANT:** Cumulus releases may introduce breaking changes between minor versions (e.g. 1.13.x -> 1.14.0), so make sure to consult the release notes for migration steps.
+**IMPORTANT:** Cumulus releases may introduce breaking changes between minor versions (e.g., 1.13.x -> 1.14.0), so it is critical that you consult the release notes for migration steps.
 
-## Migrating to a new version
+## Migrating to a New Version
 
-When breaking changes have been introduced, the Cumulus Core team will publish instructions on migrating from one version to another.
+When breaking changes have been introduced, the Cumulus Core team will publish instructions on migrating from one version to another.  Detailed release notes with migration instructions (if any) for each release can be found on the [Cumulus Releases] page.
```

Comment:
Should we mention the 'breaking changes' section in the CHANGELOG here?

Reply:
Ah, yes. Good call.

Reply:
How would you feel about adding a mention regarding deprecations as well? Something to the effect that this might be a good time to replace calls to deprecated functions?

Reply:
I like the idea to mention deprecations, but we need to make the distinction that's only needed if you're using functions from the packages. If you're just using TF deployments that does not affect you. Maybe a separate section about updating your code using Cumulus packages from npm?

Reply:
Definitely something noting that: 

- Your workflow lambdas that utilize published Core npm modules should always match your deployed Cumulus version to ensure compatibility
- Check the CHANGELOG for deprecation/breaking change notices